SDC.WebService/Services/ValidationService.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.Extensions.Logging;
+using SDC.WebService.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Xml;
+using System.Xml.Schema;
+
+namespace SDC.WebService.Services
+{
+    public class ValidationService : IValidationService
+    {
+        private readonly ILogger<ValidationService> _logger;
+        private List<string> _xsdFullPathFileNames;
+        private List<string> _xsdFileNames = new List<string> {
```

Comment:
Although this xsd file list is sufficient for validating FormDesign and DemogFormDesign and DataElement XML, we will still need to support SDCPackage, Map, IDR etc that require the other xsd files.